Composer/packages/adaptive-flow/src/adaptive-flow-renderer/widgets/DialogRef.tsx

```diff
@@ -3,23 +3,61 @@
 
 /** @jsx jsx */
 import { jsx } from '@emotion/core';
+import React from 'react';
 import get from 'lodash/get';
 import { LinkBtn, FixedInfo } from '@bfc/ui-shared';
-import { useContext } from 'react';
+import { useContext, useMemo } from 'react';
 import formatMessage from 'format-message';
 import { WidgetContainerProps, WidgetComponent } from '@bfc/extension-client';
+import { Icon } from 'office-ui-fabric-react/lib/Icon';
 
 import { NodeEventTypes } from '../constants/NodeEventTypes';
 import { RendererContext } from '../contexts/RendererContext';
 import { ElementWrapperTag } from '../types/PluggableComponents.types';
+import { NodeRendererContext } from '../../adaptive-flow-editor/contexts/NodeRendererContext';
 
 export interface DialogRefCardProps extends WidgetContainerProps {
   dialog: string | object;
 }
 
-export const DialogRef: WidgetComponent<DialogRefCardProps> = ({ id, onEvent, dialog }) => {
+export const DialogRef: WidgetComponent<DialogRefCardProps> = ({ id, onEvent, dialog, data }) => {
   const { ElementWrapper } = useContext(RendererContext);
+  const { dialogs, topics } = useContext(NodeRendererContext);
```

Comment:
@yeze322 Can you take a look at this and let me know your thoughts?

I wanted to do 2 things:
1. Use the dialog's display name instead of its id
2. Differentiate from dialogs and topics

This is your domain so I want to make sure that this approach is fine, or work with you to do something better.